_core/smart/neon-node.md

```diff
@@ -39,52 +39,41 @@ Building the project on ARM:
 cargo build --features "aarch64" --no-default-features
```

Comment:
Seems unrelated to this PR, but if we are providing testnet specific instructions on running from source, this is far simpler:

```bash
$ cargo testnet <path-to-config.toml>
```

`cargo build` will end up building everything, whereas you just want to build a single binary (equivalent of `cargo build --bin stacks-testnet` in the testnet dir).

I also see you have that invocation later in the doc, so this _will_ work, but might take longer.

Reply:
We may also want to include instructions for running an optimized binary:

```bash
$ cd testnet; cargo build --release --bin stacks-testnet; cd ..
$ /target/release/stacks-testnet <path-to-config.toml>
```

Reply:
Great suggestions! I've removed the `cargo build` step, and added a section for an optimized binary.